test/e2e/usersignup_test.go

```diff
@@ -136,6 +136,33 @@ func (s *userSignupIntegrationTest) TestAutomaticApproval() {
 	})
 }
 
+func (s *userSignupIntegrationTest) TestNotificationCreatedOKWhenUserProvisioned() {
+	// given
+	s.Host().UpdateToolchainConfig(testconfig.AutomaticApproval().Enabled(true))
+
+	// when
+	userSignup, _ := NewSignupRequest(s.T(), s.Awaitilities).
+		Username("bartsimpson").
+		Email("bartsimpson@redhat.com").
+		RequireConditions(ConditionSet(Default(), ApprovedAutomatically())...).
+		Execute().Resources()
+
+	// then
+	userSignup, err := s.Host().WaitForUserSignup(userSignup.Name,
+		wait.UntilUserSignupHasConditions(ConditionSet(Default(), ApprovedAutomatically())...),
+		wait.UntilUserSignupHasStateLabel(toolchainv1alpha1.UserSignupStateLabelValueApproved))
+	require.NoError(s.T(), err)
+
+	notifications, err := s.Host().WaitForNotifications(userSignup.Name, toolchainv1alpha1.NotificationTypeProvisioned, 1)
+	require.NoError(s.T(), err)
+
+	VerifyResourcesProvisionedForSignup(s.T(), s.Awaitilities, userSignup, "deactivate30", "base")
```

Comment:
we don't need to verify the created resources - let's focus only on the notification.